src/components/ProjectIndex/ProjectIndex.js

```diff
@@ -1,11 +1,14 @@
 import { useSelector, connect } from 'react-redux'
 import { useHistory } from 'react-router-dom'
+
 import { useProjectList } from '../Editor/Hooks/useProjectList'
 import { useRequiresUser } from '../Editor/Hooks/useRequiresUser'
+import ProjectHeader from '../ProjectHeader/ProjectHeader'
 import ProjectListItem from '../ProjectListItem/ProjectListItem'
 import Button from '../Button/Button'
-import { createOrUpdateProject } from '../../utils/apiCallHandler';
+import { createOrUpdateProject } from '../../utils/apiCallHandler'
 import { defaultPythonProject } from '../../utils/defaultProjects'
+import { NewFileIcon } from '../../Icons';
```

Comment:
Since we are reusing, might be worth renaming this to something a bit more generic e.g. `PlusIcon`

Reply:
Yep, makes sense! Done in [46e1687](https://github.com/RaspberryPiFoundation/editor-ui/pull/314/commits/46e1687230e6bd743b6c902680cea7f24181f129)